mmv1/products/dataproc/Batch.yaml

```
@@ -0,0 +1,504 @@
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'milliDcuSeconds'
+            description: |
+              DCU (Dataproc Compute Units) usage in (milliDCU x seconds)
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'shuffleStorageGbSeconds'
+            description: |
+              Shuffle storage usage in (GB x seconds)
+            output: true
+      - !ruby/object:Api::Type::NestedObject
+        name: 'currentUsage'
+        description: |
+          Approximate workload resource usage, calculated when the workload completes.
+        output: true
+        properties:
+          - !ruby/object:Api::Type::String
+            name: 'milliDcu'
+            description: |
+              Milli (one-thousandth) Dataproc Compute Units (DCUs).
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'shuffleStorageGb'
+            description: |
+              Shuffle Storage in gigabytes (GB).
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'milliDcuPremium'
+            description: |
+              Milli (one-thousandth) Dataproc Compute Units (DCUs) charged at premium tier
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'shuffleStorageGbPremium'
+            description: |
+              Shuffle Storage in gigabytes (GB) charged at premium tier.
+            output: true
+          - !ruby/object:Api::Type::String
+            name: 'snapshotTime'
+            description: |
+              The timestamp of the usage snapshot.
+            output: true
+  - !ruby/object:Api::Type::Enum
+    name: 'state'
+    description: |
+      The state of the batch.
+    output: true
+    values:
+      - :STATE_UNSPECIFIED
+      - :PENDING
+      - :RUNNING
+      - :CANCELLING
+      - :CANCELLED
+      - :SUCCEEDED
+      - :FAILED
+  - !ruby/object:Api::Type::String
+    name: 'stateMessage'
+    output: true
+    description: |
+      Batch state details, such as a failure description if the state is FAILED.
+  - !ruby/object:Api::Type::String
+    name: 'stateTime'
+    output: true
+    description: |
+      Batch state details, such as a failure description if the state is FAILED.
+  - !ruby/object:Api::Type::String
+    name: 'creator'
+    output: true
+    description: |
+      The email address of the user who created the batch.
+  - !ruby/object:Api::Type::KeyValueLabels
+    name: 'labels'
+    description: |
+      The labels to associate with this batch.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'runtimeConfig'
+    description: |
+      Runtime configuration for the batch execution.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'version'
+        description: |
+          Version of the batch runtime.
+        default_from_api: true
+        diff_suppress_func: cloudDataprocBatchFieldsDiffSuppress
```

Could be good to have a name that specifies what field this is for, or else what the function checks (SubminorVersionDiffSuppress).

```suggestion
        diff_suppress_func: cloudDataprocBatchRuntimeConfigVersionDiffSuppress
```